### PR TITLE
podmanv2 pod subcommands

### PR DESCRIPTION
**Pull request by baude**
add pod kill, pause, restart, rm, start, stop, and unpause

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

File: cmd/podmanV2/pods/rm.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
+package pods
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/containers/libpod/cmd/podmanV2/parse"
+
```
Comment (jwhonce) [inline cmd/podmanV2/pods/rm.go]:
```suggestion
```

File: cmd/podmanV2/utils/remote.go [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package utils
+
+import (
+	"github.com/containers/libpod/cmd/podmanV2/registry"
+	"github.com/containers/libpod/pkg/domain/entities"
+)
+
+func IsRemote() bool {
+	return registry.EngineOptions.EngineMode == entities.TunnelMode
+}
```
Comment (jwhonce) [inline cmd/podmanV2/utils/remote.go]:
```suggestion
```
This code is duplicate of `registry.IsRemote()`

File: pkg/domain/infra/abi/pods.go [hunk 3]
```diff
@@ -6,7 +6,9 @@ import (
 	"context"
 
 	"github.com/containers/libpod/libpod/define"
+	"github.com/containers/libpod/pkg/adapter/shortcuts"
```
Comment (jwhonce) [inline pkg/domain/infra/abi/pods.go]: Need to copy code from `adapter` into new tree.  rm -rf in `adapter`'s future.

File: cmd/podmanV2/pods/kill.go [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+var (
+	podKillDescription = `Signals are sent to the main process of each container inside the specified pod.
+
+  The default signal is SIGKILL, or any signal specified with option --signal.`
+	killCommand = &cobra.Command{
+		Use:   "kill [flags] POD [POD...]",
+		Short: "Send the specified signal or SIGKILL to containers in pod",
+		Long:  podKillDescription,
+		RunE:  kill,
```
Comment (mheon) [inline cmd/podmanV2/pods/kill.go]: Don't we need the persistentPreRunE?

Comment (jwhonce) [reply cmd/podmanV2/pods/kill.go]: @mheon The parent pod command will do that.

Comment (baude) [reply cmd/podmanV2/pods/kill.go]: nope, inherited from the pods subcommand

Comment (mheon) [reply cmd/podmanV2/pods/kill.go]: Is this documented anywhere? Because this seems somewhat unintuitive.

**Pull request merged**